src/index.scss

```diff
@@ -36,17 +37,12 @@ $white: #fff;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
-      padding-bottom: 0.1rem;
+      padding-bottom: calc(var(--pgn-spacing-spacer-base, $spacer)* 0.1);
```

Comment:
This feels like it should be fine, I can't think of any scenario where the `$spacer` value that is defined at the top of this file *wouldn't* be `1rem` by the time this is compiled into `css`.